package.json

```diff
@@ -10,20 +10,20 @@
   "scripts": {
     "lint": "eslint src",
     "build": "npm run lint && webpack --config webpack.config.js --mode production",
-    "dev": "webpack-dev-server --host 0.0.0.0 --inline --progress --config webpack.config.js --mode development"
```

Comment:
The `--inline` argument went away with the newest version of `webpack-dev-server`.